gap/SnAnBB.gi

```diff
@@ -18,100 +18,917 @@
 ##  The code is based upon the algorithm presented in the paper [BLGN+03].
 ##
 ##
-# DeclareInfoClass( "InfoRecSnAn" );
```

Comment:
Please do not overwrite the existing BB code -- add yours to a new file. So that one can compare the two side by side, benchmark them against each other etc.

Reply:
Argh, only after spending half an hour or more on this did I realize that THIS IS NOT YOUR NEW CODE, you only edited the existing stuff. :-(

I'd really prefer that in a separate PR, clearly marked as such, so that I don't waste time combing through code that is meant to be replaced. Also, I really question the usefulness of reformatting this code; it breaks `git blame` which otherwise is an important debugging tool.

I'll leave my comment in, even though I don't expect anybody to apply them to this old code.

Reply:
Argh, sorry. Will do next time. I called the commit "Uncomment most functions in gap/SnAnBB.gi" but then ofc the standard "diff view" on github doesn't show the commit messages. :/